test_runner/regress/test_branching.py

```diff
@@ -307,15 +307,17 @@ def try_branch():
         log.info(first)
         log.info(second)
 
-        (succeeded, failed) = (first, second) if isinstance(second, Exception) else (second, first)
-        assert isinstance(failed, Exception)
-        assert isinstance(succeeded, Dict)
-
-        # there's multiple valid status codes:
-        # - Timeline x/y already exists
-        # - whatever 409 response says, but that is a subclass of PageserverApiException
-        assert isinstance(failed, PageserverApiException)
-        assert succeeded["state"] == "Active"
+        # Both requests may succeed, or one may fail:
```

Comment:
@koivunej It wasn't completely obvious to me what this test is exercising -- the change I've made keeps it working, but I'm not sure how much we're proving.

The test switches off the upload pause failpoint before waiting for the subsequent branch creations to run, so it's not really proving that the  branch creation waits for the original timeline to be remotely persistent.

Reply:
Okay -- on balance I think we can remove this one.  11f667e52